src/main/frontend/extensions/srs.cljs

```diff
@@ -708,10 +708,7 @@
          [:div
           [:p (t :flashcards/modal-welcome-desc-1)]
           [:img.my-4 {:src "https://docs.logseq.com/assets/2021-07-22_22.28.02_1626964258528_0.gif"}]
-          [:p (t :flashcards/modal-welcome-desc-2)
```

Comment:
We should be passing in links as an argument but unfortunately we can't. Please revert these changes for reasons mentioned in `:on-boarding/android-intro-dropsync` and add the TODO